cumulus/parachains/runtimes/glutton/glutton-kusama/src/lib.rs

```diff
@@ -184,6 +193,25 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 
 impl parachain_info::Config for Runtime {}
 
+impl cumulus_pallet_aura_ext::Config for Runtime {}
+
+impl pallet_timestamp::Config for Runtime {
+	type Moment = u64;
+	type OnTimestampSet = Aura;
+	// TODO: set this to 0 so that anyone can produce blocks at anytime?
+	type MinimumPeriod = ConstU64<{ SLOT_DURATION / 2 }>;
+	type WeightInfo = weights::pallet_timestamp::WeightInfo<Runtime>;
+}
+
+impl pallet_aura::Config for Runtime {
+	type AuthorityId = AuraId;
+	type DisabledValidators = ();
+	type MaxAuthorities = ConstU32<100_000>;
+	type AllowMultipleBlocksPerSlot = ConstBool<false>;
+	#[cfg(feature = "experimental")]
+	type SlotDuration = pallet_aura::MinimumPeriodTimesTwo<Self>;
```

Comment:
For async backing, I think it needs to be this. However maybe this PR is just to add Aura and otherwise keep behavior the same.
```suggestion
	type SlotDuration = ConstU64<SLOT_DURATION>;
```

Reply:
The idea was to just add aura with this pr. I'll follow up with a pr containing async backing compatibility changes.